tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -2427,21 +2338,18 @@ TEST (tensorStreamTest, audioS8)
   EXPECT_EQ (g_test_data.mem_blocks, 1U);
   EXPECT_EQ (g_test_data.received_size, 500U);
 
-  /** check caps name */
-  EXPECT_STREQ (g_test_data.caps_name, "other/tensor");
-
   /** check timestamp */
   EXPECT_FALSE (g_test_data.invalid_timestamp);
 
   /** check tensor config for audio */
-  EXPECT_TRUE (gst_tensor_config_validate (&g_test_data.tensor_config));
-  EXPECT_EQ (g_test_data.tensor_config.info.type, _NNS_INT8);
-  EXPECT_EQ (g_test_data.tensor_config.info.dimension[0], 1U);
-  EXPECT_EQ (g_test_data.tensor_config.info.dimension[1], 500U);
-  EXPECT_EQ (g_test_data.tensor_config.info.dimension[2], 1U);
-  EXPECT_EQ (g_test_data.tensor_config.info.dimension[3], 1U);
-  EXPECT_EQ (g_test_data.tensor_config.rate_n, 16000);
```

Comment:
gtest will complain about the sign v.s. unsigned compare.
`16000U -> 16000`
`1U -> 1`